src/pages/artist-dashboard-page/store/sagas.js

```diff
@@ -27,12 +25,15 @@ function* fetchDashboardAsync(action) {
       currentUserId: account.user_id,
       // TODO: This only supports up to 500, we need to redesign / paginate
       // the dashboard
-      limit: account.track_count
+      // limit: account.track_count,
```

Comment:
Had to remove limit because we don't know how many unlisted tracks a user can have. I can update it to be 500 if that makes sense too.

Reply:
Default server side is 500? i think we can just leave no arg here then! 👍